hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieMergeHandle.java

```diff
@@ -399,9 +429,26 @@ protected void writeIncomingRecords() throws IOException {
     }
   }
 
+  private void setCDCStatIfNeeded(Option<AppendResult> cdcResult, HoodieWriteStat stat) {
```

Comment:
Move this also to under HoodieCDCLogger?

Reply:
ok